benchmark/syclblas/extension/omatcopy.cpp

```diff
@@ -0,0 +1,155 @@
+/**************************************************************************
```

Comment:
What about the `omatcopy2` benchmark? Don't we need it?

Reply:
These benchmarks have been under discussion for some time. We decided to include benchmarks for `omatcopy` and other operators in the PR that builds upon it because cuBLAS and rocBLAS have a similar operator. However, we later realized that comparing `omatcopy` may not be appropriate due to slight differences in the operator. Therefore, we also omitted it for cuBLAS and rocBLAS.

As for `omatcopy2`, it is different from the others because there is no suitable point of comparison. Hence, we did not include it. Should we consider its inclusion?

Reply:
Probably, we should include it because it is present in oneMKL.